kubectl.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path="typings/globals/node/index.d.ts" />
 "use strict"
 
-import tl = require('vsts-task-lib');
+import tl = require('vsts-task-lib/task');
```

Comment:
this was preventing the mock from getting called (real impl was getting called during test)